src/dhtproto/client/internal/NodeHashRanges.d

```diff
@@ -85,15 +86,15 @@ public final class NodeHashRanges : NodeHashRangesBase
 
     ***************************************************************************/
 
-    public void getFromNode ( hash_t h, ref NodeHashRange[] node_hash_ranges,
-        UseNodeDg use_node,
+    public void getFromNode ( hash_t h,
+        VoidBufferAsArrayOf!(NodeHashRange) node_hash_ranges, UseNodeDg use_node,
```

Comment:
Nice that this doesn't leak anywhere to the user :+1:

Reply:
Yeah, the user has no idea about the node hash range registry. Purely internal client stuff.